config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=17.9
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=17.9.0.1
+VERSION_LONG=17.9.0.2
```

Comment:
I see you started to adjust your PR titles to be zero-based (beta 0, beta 1, beta 2) to make it less surprising on them matching the version used 👍 👏  (I'd also be ok with a PR title of "Merge 17.9.0.2 into develop" fwiw which avoids confusion as if you're zero- or one-based when talking about "beta N" 🤷 )